sklearn/metrics/cluster/tests/test_unsupervised.py

```diff
@@ -183,6 +184,14 @@ def test_silhouette_nonzero_diag(dtype):
     with pytest.raises(ValueError, match='contains non-zero'):
         silhouette_samples(dists, labels, metric='precomputed')
 
+@pytest.mark.parametrize('store_order', ('row-major', 'col-major'))
+def test_silhouette_sparse_input(store_order):
+    # Ensure that silhouette_samples works for sparse matrix inputs
```

Comment:
We are trying to push for use docstrings instead of comments to document test.

```suggestion
    """Ensure that silhouette_samples works for sparse matrix inputs"""
```